Classes/Domain/Model/Redirect.php

```diff
@@ -11,7 +13,11 @@
  * source code.
  */
 
+use DateTime;
```

Comment:
I don't do that, importing top-level names is annoying IMHO. I know that PhpStorm tends to complain, but that's easily disabled. For me, that's a one-char gain that needs to be paid for with confusion ("What, a custom DateTime type?")